src/aatoolbox/datasources/fewsnet/fewsnet.py

```diff
@@ -27,223 +33,335 @@
     "HFIC/{region_code}/{region_name}{YYYY}{MM}.zip"
 )
 
-_VALID_REGION_NAMES = [
-    "caribbean-central-america",
-    "central-asia",
-    "east-africa",
-    "southern-africa",
-    "west-africa",
-]
-_VALID_REGION_CODES = ["LAC", "EA", "CA", "SA", "WA"]
-
-
-def download_fewsnet(
-    date_pub: Union[date, str],
-    iso2: str,
-    region_name: Literal[
-        "caribbean-central-america",
-        "central-asia",
-        "east-africa",
-        "southern-africa",
-        "west-africa",
-    ],
-    region_code: Literal["LAC", "EA", "CA", "SA", "WA"],
-    output_dir: Union[Path, str],
-    use_cache=True,
-) -> Path:
-    """
-    Retrieve the raw fewsnet data.
+_VALID_PROJECTION_PERIODS_LITERAL = Literal["CS", "ML1", "ML2"]
+_VALID_PROJECTION_PERIODS = ["CS", "ML1", "ML2"]
```

Comment:
What wasn't working for `__args__`? I got it to work fine just ignoring using `mypy`.

Reply:
yeah I first thought so as well, it is importing it fine. But then when it is actually using the `__args__` in the tests, it got an error for 3.6 that it couldnt find `__args__`.. Dont really understand why but didnt want to waste more time on it.

Reply:
Oh very strange!

Reply:
Hmm the last answer didnt work either (error that Literal cannot accept tuples). Now used an enum but think it is overcomplicating things a bit for what it should do. Is this the way you imagined to use enum here?

Reply:
I think it looks good with enum! My only suggestion would be instead of doing `projperiods = ValidProjectionPeriods._value2member_map_` (aha where did you find this method) you can do:
```
projperiods =  [pp.value for pp in ValidProjectionPeriods]
```
and maybe even just merge it with the next line